doc/doxygen/src/ForManagers.dox

```diff
@@ -211,18 +203,6 @@
  *   tasks. For example, loading multiple AOVs for a render, or
  *   determining data dependencies when transferring assets.
  *
- * - If appropriate, implement
- *   @ref openassetio.managerApi.ManagerInterface.ManagerInterface.setRelatedReferences
- *   "setRelatedReferences" to update any internal relationships that
- *   may be affected by the registered change.
- *
- * @note It is currently in flux as to whether relationship creation
- * should be a hard requirement of the API, or considered hints as set
- * by the host. There is even question as to the validity of the API
- * allowing sets at all, please contribute to the discussion here if you
- * have any opinion on the matter:
- * https://github.com/OpenAssetIO/OpenAssetIO/discussions/18
```

Comment:
[consider][minor] Perhaps worth retaining this link for now?

Reply:
I was pondering this - and thought it might be worth leaving out for now as we definitely have use cases that require a specific method (eg. connecting a render to the existing, un-changed, assetized script that produced it).